### DISCUSSION TITLE
Which database services are you deploying on AWS?

### DISCUSSION THREAD

**Discussion by dangdennis** [encoredev/encore#272] [Help] [answered]
I'd like to know how you're deploying the postgres databases onto AWS.

1. Is it pg deployed on a Kubernetes cluster? Or can I choose to use aws aurora (serverless or dedicated)?
2. How do you handle multi-region deploys and read-replicas? I see this is part of your enterprise billing plan only so far.
3. If we have multi-region deployments, how do we select the primary region / write database?

**Comment by eandre** [comment, marked as answer]
Hey @dangdennis, great questions.

Postgres on AWS currently provisions RDS, but we've looked at supporting Aurora as well.

As for multi-region deployments we've discussed making it more broadly available. We're still iterating on how they work, so happy to discuss if you have specific requirements and we can make sure your use case is supported. The general approach is multi-region load balancing for the application and then deploying read replicas to the same zones. We can configure the primary region for the write primary on a per-environment basis.

**Reply by dangdennis** [reply]
I just found the bit of docs that would've partially answered my question too. Oops and thank you! Your solution matches what I had hoped 😄 . No specific requirements as of yet. 

https://encore.dev/docs/deploy/environments

```
Production environments are provisioned for maximum reliability, availability and scalability. Databases are provisioned as proper, managed databases with automatic backups. Your backend code runs with auto-scaling to match your traffic requirements.

Development environments are provisioned for simplicity, cost efficiency and speed. The databases are provisioned with persistent disks using Kubernetes, to offer reasonable durability and scalability, suitable for the most development needs.
```
